src/tests/suite/dfs_test.c

```diff
@@ -674,9 +674,14 @@ dfs_test_hole_mgmt(void **state)
 	/** reset */
 	memset(rbuf[0], '-', buf_size + 100);
 
+	MPI_Barrier(MPI_COMM_WORLD);
```

Comment:
Is this barrier really required ?

Reply:
yes because rank 0 couldrace ahead and punch the file and the others are still lagging behind doing the read.

Reply:
Eh eh, I should have unhide previous code before asking !!... ;-)